conda/recipes/numbast_extensions/meta.yaml

```diff
@@ -19,36 +19,17 @@ source:
 build:
   script:
     - {{ PYTHON }} -m pip install . -vv
-  ignore_run_exports_from:
-    - {{ compiler('cuda') }}
 
 requirements:
   build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}
-    - cmake >=3.28
-    - pybind11
-    - cffi
-    - clangdev >=18
-    - ninja
-    - ast_canopy
-    - numbast
-    - sysroot_{{ target_platform }} 2.17
-  host:
-    - cuda-toolkit {{ cuda_compiler_version }}
-    - cuda-python {{ cuda_compiler_version }}
-    - cuda-version {{ cuda_compiler_version }}
     - python
     - pip
-    - scikit-build-core
   run:
+    - cuda-version {{ cuda_compiler_version }}
+    - cuda-cudart-dev
     - python
     - numba >=0.59
```

Comment:
It's a bit odd that the different packages have different numba version requirements. Is that accurate? 0.58 vs. 0.59?

Reply:
I don't think `ast_canopy` should depend on numba at all.